deployments/base/deployment.yaml

```diff
@@ -0,0 +1,64 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  annotations:
+  name: jwtinspect
+  labels:
+    app: jwtinspect
+    version: v1
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: jwtinspect
+      version: v1
```

Comment:
question: do we need version: v1 label here. what is the intention?

Reply:
We don't, so I can just remove it